recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "aiopg" %}
-{% set version = "1.0.0" %}
+{% set version = "1.3.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 32da3c9c73c248c715883fbb0876d66015d9fb6b4946058eb9fd40e86cfff9b4
+  sha256: 23f9e4cd9f28e9d91a6de3b4fb517e8bed25511cd954acccba9fe3a702d9b7d0
 
 build:
   number: 0
```

Comment:
We probably should avoid building `noarch` packages as much as possible. 

https://github.com/AnacondaRecipes/aiopg-feedstock/blob/7fc0166a16ab43c920e620a9c95251703efadb42/recipe/meta.yaml#L14

Reply:
Yep, and there should be `skip: True  # [py<36]`, see https://github.com/aio-libs/aiopg/blob/7f40980694085bb176d9ef2a5529f026941183db/setup.py#L61

Reply:
removed no arch, and added skip